plugins/woocommerce-admin/client/task-lists/setup-task-list/components/task-list-item.tsx

```diff
@@ -71,13 +71,17 @@ export const TaskListItem: React.FC< TaskListItemProps > = ( {
 				}
 			);
 
-			const onClickActions = () => {
+			const onClick = ( e: React.MouseEvent< HTMLButtonElement > ) => {
+				if ( ( e.target as HTMLElement ).tagName === 'A' ) {
```

Comment:
Just wondering, is this because the <a> elements in the task list have their own event handlers?

Reply:
It's because we want to navigate to the link when the user clicks on it. We don't want to trigger onClick for the link. It's not related to this PR. I just moved the code below to the onClick handler so that it's easier to read and disable the click event.